packages/bits/spec/components/sorter/sorter.e2e.ts

```diff
@@ -63,5 +64,39 @@ describe("USERCONTROL Sorter >", () => {
             });
         });
 
+        describe("key navigation >", () => {
+            it("should close sorter menu when navigating from it by TAB key", async () => {
+                await sorter.click();
+                expect(await sorter.isPopupDisplayed()).toBe(true);
+                await Helpers.pressKey(Key.TAB);
+                expect(await sorter.isPopupDisplayed()).toBe(false);
+            });
+            // TODO Change this test in the scope of NUI-6132
+            it("should close sorter menu by ESCAPE key, open by ENTER, and select first item", async () => {
+                await sorter.click();
+                expect(await sorter.isPopupDisplayed()).toBe(true);
+                await Helpers.pressKey(Key.ESCAPE);
+                expect(await sorter.isPopupDisplayed()).toBe(false);
+                await Helpers.pressKey(Key.ENTER);
+                await Helpers.pressKey(Key.ARROW_DOWN);
+                await Helpers.pressKey(Key.ENTER);
+                expect(await sorter.getCurrentValue()).toBe("Title");
+            });
+            // TODO Change this test in the scope of NUI-6132
+            it("should select sort items by keyboard", async () => {
+                await sorter.click();
+                await Helpers.pressKey(Key.ARROW_DOWN);
+                await Helpers.pressKey(Key.ARROW_DOWN);
```

Comment:
You can use the second arg of the function to specify how many times to click the key

Reply:
this way you won't be duplicating the lines of code